src/components/Widget/GoogleReCaptchaWidget.jsx

```diff
@@ -12,7 +12,7 @@ const GoogleReCaptchaWidget = ({ onVerify, GoogleReCaptcha }) => {
 
   return (
     <GoogleReCaptchaProvider
-      reCaptchaKey={process.env.RAZZLE_RECAPTCHA_KEY}
+      reCaptchaKey={sitekey}
```

Comment:
@mamico i cannot understand from where is now readed the RAZZLE_HCAPTCHA_KEY or RAZZLE_RECAPTCHA_KEY value..

Reply:
@giuliaghisini in Plone settings. Serialized by restapi: https://github.com/collective/collective.volto.formsupport/#block-serializer